doc/migration/version-upgrade-guide.md

```diff
@@ -1058,6 +1058,54 @@ RUN ansible-galaxy collection install -r ${HOME}/requirements.yml \
 
 ## Unreleased (Master Branch)
 
+### modules
+
+- Ensure that the following `require` modules and `replace` directives with the specific versions are present in your `go.mod` file:
+
+```
+require (
+	github.com/operator-framework/operator-sdk v0.0.0
+	github.com/spf13/pflag v1.0.5
+	k8s.io/api v0.17.4
+	k8s.io/apimachinery v0.17.4
+	k8s.io/client-go v12.0.0+incompatible
+	sigs.k8s.io/controller-runtime v0.5.2
+)
+
+// Pinned to kubernetes-1.17.4
+replace (
+	k8s.io/api => k8s.io/api v0.17.4
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.17.4
+	k8s.io/apimachinery => k8s.io/apimachinery v0.17.4
+	k8s.io/apiserver => k8s.io/apiserver v0.17.4
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.17.4
+	k8s.io/client-go => k8s.io/client-go v0.0.0-20191016111102-bec269661e48
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.17.4
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.17.4
+	k8s.io/code-generator => k8s.io/code-generator v0.17.4
+	k8s.io/component-base => k8s.io/component-base v0.17.4
+	k8s.io/cri-api => k8s.io/cri-api v0.17.4
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.17.4
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.17.4
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.17.4
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.17.4
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.17.4
+	k8s.io/kubectl => k8s.io/kubectl v0.17.4
+	k8s.io/kubelet => k8s.io/kubelet v0.17.4
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.17.4
+	k8s.io/metrics => k8s.io/metrics v0.17.4
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.17.4
+)
+
+replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309 // Required by Helm
```

Comment:
I don't think this `replace` is required after the Helm bump to 3.1. But I think this one is now:
```suggestion
replace github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d // Required by Helm
```

Reply:
We. need to keep the docker one. See:

```
go: github.com/helm/helm-2to3@v0.2.0 requires
	helm.sh/helm/v3@v3.0.0 requires
	github.com/deislabs/oras@v0.7.0 requires
	github.com/docker/docker@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
```

Reply:
I could remove it from go.mod of SDK after upgrade the `github.com/helm/helm-2to3`to `v0.4.1`. However, I understand that we will able to check and change if possible the scaffold files just in a follow-up PR.

Reply:
To keep things aligned, we should bump to the latest helm-2to3 that depends on helm 3.1. Looks like that would be the latest release (`v0.5.1`)

Reply:
Make sense .. we need to upgrade it and then add `github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d` in the scaffolds and migration guide.

Reply:
Done 👍